src/test/java/teammates/test/cases/ui/InstructorEditInstructorFeedbackPageActionTest.java

```diff
@@ -74,11 +75,12 @@ public void testExecuteAndPostProcess() {
         assertEquals(Const.ViewURIs.INSTRUCTOR_FEEDBACK_SUBMISSION_EDIT + "?error=false&user=" + instructor.googleId,
                      showPageResult.getDestinationWithParams());
         assertEquals("", showPageResult.getStatusMessage());
-        AssertHelper.assertLogMessageEquals("TEAMMATESLOG|||instructorEditInstructorFeedbackPage|||instructorEditInstructorFeedbackPage"
-                + "|||true|||Instructor|||IEIFPTCourseinstr|||IEIFPTCourseinstr|||IEIFPTCourseintr@course1.tmt|||"
-                + "Moderating feedback session for instructor (" + moderatedInstructor.email + ")<br>"
-                + "Session Name: Another feedback session<br>Course ID: IEIFPTCourse|||"
-                + "/page/instructorEditInstructorFeedbackPage",
+        AssertHelper.assertLogMessageEquals(
+                "TEAMMATESLOG|||instructorEditInstructorFeedbackPage|||instructorEditInstructorFeedbackPage"
+                    + "|||true|||Instructor|||IEIFPTCourseinstr|||IEIFPTCourseinstr|||IEIFPTCourseintr@course1.tmt|||"
+                    + "Moderating feedback session for instructor (" + moderatedInstructor.email + ")<br>"
+                    + "Session Name: Another feedback session<br>Course ID: IEIFPTCourse|||"
+                    + "/page/instructorEditInstructorFeedbackPage",
                 editInstructorFpAction.getLogMessage());
```

Comment:
Inconsistency: the lines with `+` are indented 4 spaces to separate the first and second arguments, but elsewhere, 8 spaces are used (e.g., below at `InstructorEditStudentFeedbackPageActionTest.java:150`